src/swarm/util/RecordStream.d

```diff
@@ -178,13 +178,15 @@ public struct Record
 
     ***************************************************************************/
 
-    private void deserialize ( Console.Input stream )
```

Comment:
`Returns:` doc.

Reply:
By the way, is there a reason why this `deserialize` overload is private but the other one is public? (Not sure if I noticed that in the previous PR.)

Reply:
Yeah, because it's a rather ugly hack. I basically say "We only accept Tango's class stdin", while a more carefully designed code would accept any `BufferedInput`.

Reply:
Maybe [this](https://github.com/sociomantic-tsunami/ocean/blob/6d387eaf835d85c88dbc6a6b1b1090a0863a26b5/src/ocean/io/Console.d#L105) would help

Reply:
Hmm... it appears that the idea was that `deserialize` provides feedback about EOF by throwing. (See the `catch` in `fiberMethod`.)

Reply:
Yes, that's correct. However I don't know how to cleanly do it with an exception. Should we have one exception per record ? Should `fiberMethod` provide the exception ? Also a `throw` seems rather wasteful compared to a `return false;`. So I agree that this mix of semantic is sub-ideal, but IMO it's a temporary situation.